commands/audit/sca/npm/npm_test.go

```diff
@@ -122,3 +124,21 @@ func TestIgnoreScripts(t *testing.T) {
 	_, _, err := BuildDependencyTree(params)
 	assert.NoError(t, err)
 }
+
+// This test checks that the tree construction is skipped when the project is not installed and the user prohibited installation
+func TestSkipBuildDepTreeWhenInstallForbidden(t *testing.T) {
```

Comment:
Can you add a test we requested to skip but the project is already installed? in this case we don't expect an error

Reply:
added for yarn and npm